### DISCUSSION TITLE
How to use the VGA screen?

### DISCUSSION THREAD

**Discussion by xtexx** [logisim-evolution/logisim-evolution#800] [Q&A] [answered]
What is the BUS?

I am designing a circult made by 74 gates to reading data from EEPROM and display the image to th VGA screen.

How can I use the VGA screen? 

The Library Reference is not fully.

**Comment by maehne** [comment, marked as answer]
The VGA screen component can only be used in conjunction with the other components from the System On a Chip library. It is not meant to be directly driven by logic gate circuit. It behaves as a memory-mapped device on the connected "SoC bus simulator" component. Typically, you would initiate these writes to the pixel buffer from an assembler program executed by the RISC-V or Nios-IIs simulator.

Indeed, the SoC library still lacks documentation, because we have used it so far only internally in the context of a SoC design introduction course.
